analyzers/tests/SonarAnalyzer.UnitTest/Rules/DisposableNotDisposedTest.cs

```diff
@@ -36,6 +36,12 @@ public class DisposableNotDisposedTest
 
 #if NET
 
+        [TestMethod]
+        public void DisposableNotDisposed_CSharp8() =>
```

Comment:
The previous test “ DisposableNotDisposed()” also starts from C# 8, so this makes two methods have the same code and different names —> confusing IMO

I remember our conversation on this, I did not have this problem in mind, lets find a better name

Reply:
I moved all test cases from `DisposableNotDisposed` which aren't supported by C# 7 to `DisposableNotDisposed.CSharp8`
Then set the original `DisposableNotDisposed` to use C# 7 for testing.